databases/week2/week2-databases-homework.sql

```diff
@@ -0,0 +1,89 @@
+Part 1: Working with tasks
+Add a task with these attributes: title, description, created, updated, due_date, status_id, user_id
+insert into task 
+(title, description, created, updated, due_date, status_id, user_id) 
+values 
+('listening audio book', 'learning Danish during physical working', '2020-02-07 05:00:00', '2020-02-07 10:00:00', '2020-02-07 12:00:00', 2, 1);
+Change the title of a task
+update task set title = 'listening to et liv en tid book' where id = 36;
+Change a task due date
+update task set due_date = '2020-01-07 12:00:00' where id = 36;
+Change a task status
+update task set status_id = 1 where id = 36;
+Mark a task as complete
+update task set status_id = 3 where id = 36;
+Delete a task
+delete from task where id = 36;
+select count(*), created from task group by wher month(created)=10;
```

Comment:
A spelling error is there
select count(*), created from task group by `wher` month(created)=10;